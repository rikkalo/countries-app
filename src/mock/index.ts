export const countriesMock = [
  {
    name: "Botswana",
    topLevelDomain: [".bw"],
    alpha2Code: "BW",
    alpha3Code: "BWA",
    callingCodes: ["267"],
    capital: "Gaborone",
    altSpellings: ["BW", "Republic of Botswana", "Lefatshe la Botswana"],
    region: "Africa",
    subregion: "Southern Africa",
    population: 2141206,
    latlng: [-22, 24],
    demonym: "Motswana",
    area: 582000,
    gini: 61,
    density: 4,
    timezones: ["UTC+02:00"],
    borders: ["NAM", "ZAF", "ZMB", "ZWE"],
    nativeName: "Botswana",
    numericCode: "072",
    currencies: [
      {
        code: "BWP",
        name: "Botswana pula",
        symbol: "P"
      }
    ],
    languages: [
      {
        iso639_1: "en",
        iso639_2: "eng",
        name: "English",
        nativeName: "English"
      },
      {
        iso639_1: "tn",
        iso639_2: "tsn",
        name: "Tswana",
        nativeName: "Setswana"
      }
    ],
    translations: {
      de: "Botswana",
      es: "Botswana",
      fr: "Botswana",
      ja: "ボツワナ",
      it: "Botswana",
      br: "Botsuana",
      pt: "Botsuana",
      nl: "Botswana",
      hr: "Bocvana",
      fa: "بوتسوانا"
    },
    flag: "https://restcountries.eu/data/bwa.svg",
    regionalBlocs: [
      {
        acronym: "AU",
        name: "African Union",
        otherAcronyms: [],
        otherNames: [
          "الاتحاد الأفريقي",
          "Union africaine",
          "União Africana",
          "Unión Africana",
          "Umoja wa Afrika"
        ]
      }
    ],
    cioc: "BOT"
  },
  {
    name: "Algeria",
    topLevelDomain: [".dz"],
    alpha2Code: "DZ",
    alpha3Code: "DZA",
    callingCodes: ["213"],
    capital: "Algiers",
    altSpellings: ["DZ", "Dzayer", "Algérie"],
    region: "Africa",
    subregion: "Northern Africa",
    population: 40400000,
    latlng: [28, 3],
    demonym: "Algerian",
    area: 2381741,
    gini: 35.3,
    timezones: ["UTC+01:00"],
    borders: ["TUN", "LBY", "NER", "ESH", "MRT", "MLI", "MAR"],
    nativeName: "الجزائر",
    numericCode: "012",
    currencies: [
      {
        code: "DZD",
        name: "Algerian dinar",
        symbol: "د.ج"
      }
    ],
    languages: [
      {
        iso639_1: "ar",
        iso639_2: "ara",
        name: "Arabic",
        nativeName: "العربية"
      }
    ],
    translations: {
      de: "Algerien",
      es: "Argelia",
      fr: "Algérie",
      ja: "アルジェリア",
      it: "Algeria",
      br: "Argélia",
      pt: "Argélia",
      nl: "Algerije",
      hr: "Alžir",
      fa: "الجزایر"
    },
    flag: "https://restcountries.eu/data/dza.svg",
    regionalBlocs: [
      {
        acronym: "AU",
        name: "African Union",
        otherAcronyms: [],
        otherNames: [
          "الاتحاد الأفريقي",
          "Union africaine",
          "União Africana",
          "Unión Africana",
          "Umoja wa Afrika"
        ]
      }
    ],
    cioc: "ALG"
  },
  {
    name: "Chad",
    topLevelDomain: [".td"],
    alpha2Code: "TD",
    alpha3Code: "TCD",
    callingCodes: ["235"],
    capital: "N'Djamena",
    altSpellings: ["TD", "Tchad", "Republic of Chad", "République du Tchad"],
    region: "Africa",
    subregion: "Middle Africa",
    population: 14497000,
    latlng: [15, 19],
    demonym: "Chadian",
    area: 1284000,
    gini: 39.8,
    density: 11,
    timezones: ["UTC+01:00"],
    borders: ["CMR", "CAF", "LBY", "NER", "NGA", "SSD"],
    nativeName: "Tchad",
    numericCode: "148",
    currencies: [
      {
        code: "XAF",
        name: "Central African CFA franc",
        symbol: "Fr"
      }
    ],
    languages: [
      {
        iso639_1: "fr",
        iso639_2: "fra",
        name: "French",
        nativeName: "français"
      },
      {
        iso639_1: "ar",
        iso639_2: "ara",
        name: "Arabic",
        nativeName: "العربية"
      }
    ],
    translations: {
      de: "Tschad",
      es: "Chad",
      fr: "Tchad",
      ja: "チャド",
      it: "Ciad",
      br: "Chade",
      pt: "Chade",
      nl: "Tsjaad",
      hr: "Čad",
      fa: "چاد"
    },
    flag: "https://restcountries.eu/data/tcd.svg",
    regionalBlocs: [
      {
        acronym: "AU",
        name: "African Union",
        otherAcronyms: [],
        otherNames: [
          "الاتحاد الأفريقي",
          "Union africaine",
          "União Africana",
          "Unión Africana",
          "Umoja wa Afrika"
        ]
      }
    ],
    cioc: "CHA"
  }
];

export const countriesWithDensityMock = [
  {
    name: "Algeria",
    topLevelDomain: [".dz"],
    alpha2Code: "DZ",
    alpha3Code: "DZA",
    callingCodes: ["213"],
    capital: "Algiers",
    altSpellings: ["DZ", "Dzayer", "Algérie"],
    region: "Africa",
    subregion: "Northern Africa",
    population: 40400000,
    latlng: [28, 3],
    demonym: "Algerian",
    area: 2381741,
    gini: 35.3,
    timezones: ["UTC+01:00"],
    borders: ["TUN", "LBY", "NER", "ESH", "MRT", "MLI", "MAR"],
    nativeName: "الجزائر",
    numericCode: "012",
    currencies: [{ code: "DZD", name: "Algerian dinar", symbol: "د.ج" }],
    languages: [
      {
        iso639_1: "ar",
        iso639_2: "ara",
        name: "Arabic",
        nativeName: "العربية"
      }
    ],
    translations: {
      de: "Algerien",
      es: "Argelia",
      fr: "Algérie",
      ja: "アルジェリア",
      it: "Algeria",
      br: "Argélia",
      pt: "Argélia",
      nl: "Algerije",
      hr: "Alžir",
      fa: "الجزایر"
    },
    flag: "https://restcountries.eu/data/dza.svg",
    regionalBlocs: [
      {
        acronym: "AU",
        name: "African Union",
        otherAcronyms: [],
        otherNames: [
          "الاتحاد الأفريقي",
          "Union africaine",
          "União Africana",
          "Unión Africana",
          "Umoja wa Afrika"
        ]
      }
    ],
    cioc: "ALG",
    density: 17
  },
  {
    name: "Botswana",
    topLevelDomain: [".bw"],
    alpha2Code: "BW",
    alpha3Code: "BWA",
    callingCodes: ["267"],
    capital: "Gaborone",
    altSpellings: ["BW", "Republic of Botswana", "Lefatshe la Botswana"],
    region: "Africa",
    subregion: "Southern Africa",
    population: 2141206,
    latlng: [-22, 24],
    demonym: "Motswana",
    area: 582000,
    gini: 61,
    density: 4,
    timezones: ["UTC+02:00"],
    borders: ["NAM", "ZAF", "ZMB", "ZWE"],
    nativeName: "Botswana",
    numericCode: "072",
    currencies: [{ code: "BWP", name: "Botswana pula", symbol: "P" }],
    languages: [
      {
        iso639_1: "en",
        iso639_2: "eng",
        name: "English",
        nativeName: "English"
      },
      {
        iso639_1: "tn",
        iso639_2: "tsn",
        name: "Tswana",
        nativeName: "Setswana"
      }
    ],
    translations: {
      de: "Botswana",
      es: "Botswana",
      fr: "Botswana",
      ja: "ボツワナ",
      it: "Botswana",
      br: "Botsuana",
      pt: "Botsuana",
      nl: "Botswana",
      hr: "Bocvana",
      fa: "بوتسوانا"
    },
    flag: "https://restcountries.eu/data/bwa.svg",
    regionalBlocs: [
      {
        acronym: "AU",
        name: "African Union",
        otherAcronyms: [],
        otherNames: [
          "الاتحاد الأفريقي",
          "Union africaine",
          "União Africana",
          "Unión Africana",
          "Umoja wa Afrika"
        ]
      }
    ],
    cioc: "BOT"
  },
  {
    name: "Chad",
    topLevelDomain: [".td"],
    alpha2Code: "TD",
    alpha3Code: "TCD",
    callingCodes: ["235"],
    capital: "N'Djamena",
    altSpellings: ["TD", "Tchad", "Republic of Chad", "République du Tchad"],
    region: "Africa",
    subregion: "Middle Africa",
    population: 14497000,
    latlng: [15, 19],
    demonym: "Chadian",
    area: 1284000,
    gini: 39.8,
    density: 11,
    timezones: ["UTC+01:00"],
    borders: ["CMR", "CAF", "LBY", "NER", "NGA", "SSD"],
    nativeName: "Tchad",
    numericCode: "148",
    currencies: [
      { code: "XAF", name: "Central African CFA franc", symbol: "Fr" }
    ],
    languages: [
      {
        iso639_1: "fr",
        iso639_2: "fra",
        name: "French",
        nativeName: "français"
      },
      {
        iso639_1: "ar",
        iso639_2: "ara",
        name: "Arabic",
        nativeName: "العربية"
      }
    ],
    translations: {
      de: "Tschad",
      es: "Chad",
      fr: "Tchad",
      ja: "チャド",
      it: "Ciad",
      br: "Chade",
      pt: "Chade",
      nl: "Tsjaad",
      hr: "Čad",
      fa: "چاد"
    },
    flag: "https://restcountries.eu/data/tcd.svg",
    regionalBlocs: [
      {
        acronym: "AU",
        name: "African Union",
        otherAcronyms: [],
        otherNames: [
          "الاتحاد الأفريقي",
          "Union africaine",
          "União Africana",
          "Unión Africana",
          "Umoja wa Afrika"
        ]
      }
    ],
    cioc: "CHA"
  }
];

export const countryMock = {
  name: "Algeria",
  topLevelDomain: [".dz"],
  alpha2Code: "DZ",
  alpha3Code: "DZA",
  callingCodes: ["213"],
  capital: "Algiers",
  altSpellings: ["DZ", "Dzayer", "Algérie"],
  region: "Africa",
  subregion: "Northern Africa",
  population: 40400000,
  latlng: [28, 3],
  demonym: "Algerian",
  area: 2381741,
  gini: 35.3,
  density: 17,
  timezones: ["UTC+01:00"],
  borders: ["TUN", "LBY", "NER", "ESH", "MRT", "MLI", "MAR"],
  nativeName: "الجزائر",
  numericCode: "012",
  currencies: [
    {
      code: "DZD",
      name: "Algerian dinar",
      symbol: "د.ج"
    }
  ],
  languages: [
    {
      iso639_1: "ar",
      iso639_2: "ara",
      name: "Arabic",
      nativeName: "العربية"
    }
  ],
  translations: {
    de: "Algerien",
    es: "Argelia",
    fr: "Algérie",
    ja: "アルジェリア",
    it: "Algeria",
    br: "Argélia",
    pt: "Argélia",
    nl: "Algerije",
    hr: "Alžir",
    fa: "الجزایر"
  },
  flag: "https://restcountries.eu/data/dza.svg",
  regionalBlocs: [
    {
      acronym: "AU",
      name: "African Union",
      otherAcronyms: [],
      otherNames: [
        "الاتحاد الأفريقي",
        "Union africaine",
        "União Africana",
        "Unión Africana",
        "Umoja wa Afrika"
      ]
    }
  ],
  cioc: "ALG"
};

export const countriesWithAreaNullMock = [
  {
    name: "Botswana",
    topLevelDomain: [".bw"],
    alpha2Code: "BW",
    alpha3Code: "BWA",
    callingCodes: ["267"],
    capital: "Gaborone",
    altSpellings: ["BW", "Republic of Botswana", "Lefatshe la Botswana"],
    region: "Africa",
    subregion: "Southern Africa",
    population: 2141206,
    latlng: [-22, 24],
    demonym: "Motswana",
    area: null,
    gini: 61,
    density: 4,
    timezones: ["UTC+02:00"],
    borders: ["NAM", "ZAF", "ZMB", "ZWE"],
    nativeName: "Botswana",
    numericCode: "072",
    currencies: [
      {
        code: "BWP",
        name: "Botswana pula",
        symbol: "P"
      }
    ],
    languages: [
      {
        iso639_1: "en",
        iso639_2: "eng",
        name: "English",
        nativeName: "English"
      },
      {
        iso639_1: "tn",
        iso639_2: "tsn",
        name: "Tswana",
        nativeName: "Setswana"
      }
    ],
    translations: {
      de: "Botswana",
      es: "Botswana",
      fr: "Botswana",
      ja: "ボツワナ",
      it: "Botswana",
      br: "Botsuana",
      pt: "Botsuana",
      nl: "Botswana",
      hr: "Bocvana",
      fa: "بوتسوانا"
    },
    flag: "https://restcountries.eu/data/bwa.svg",
    regionalBlocs: [
      {
        acronym: "AU",
        name: "African Union",
        otherAcronyms: [],
        otherNames: [
          "الاتحاد الأفريقي",
          "Union africaine",
          "União Africana",
          "Unión Africana",
          "Umoja wa Afrika"
        ]
      }
    ],
    cioc: "BOT"
  },
  {
    name: "Algeria",
    topLevelDomain: [".dz"],
    alpha2Code: "DZ",
    alpha3Code: "DZA",
    callingCodes: ["213"],
    capital: "Algiers",
    altSpellings: ["DZ", "Dzayer", "Algérie"],
    region: "Africa",
    subregion: "Northern Africa",
    population: 40400000,
    latlng: [28, 3],
    demonym: "Algerian",
    area: 2381741,
    gini: 35.3,
    timezones: ["UTC+01:00"],
    borders: ["TUN", "LBY", "NER", "ESH", "MRT", "MLI", "MAR"],
    nativeName: "الجزائر",
    numericCode: "012",
    currencies: [
      {
        code: "DZD",
        name: "Algerian dinar",
        symbol: "د.ج"
      }
    ],
    languages: [
      {
        iso639_1: "ar",
        iso639_2: "ara",
        name: "Arabic",
        nativeName: "العربية"
      }
    ],
    translations: {
      de: "Algerien",
      es: "Argelia",
      fr: "Algérie",
      ja: "アルジェリア",
      it: "Algeria",
      br: "Argélia",
      pt: "Argélia",
      nl: "Algerije",
      hr: "Alžir",
      fa: "الجزایر"
    },
    flag: "https://restcountries.eu/data/dza.svg",
    regionalBlocs: [
      {
        acronym: "AU",
        name: "African Union",
        otherAcronyms: [],
        otherNames: [
          "الاتحاد الأفريقي",
          "Union africaine",
          "União Africana",
          "Unión Africana",
          "Umoja wa Afrika"
        ]
      }
    ],
    cioc: "ALG"
  },
  {
    name: "Chad",
    topLevelDomain: [".td"],
    alpha2Code: "TD",
    alpha3Code: "TCD",
    callingCodes: ["235"],
    capital: "N'Djamena",
    altSpellings: ["TD", "Tchad", "Republic of Chad", "République du Tchad"],
    region: "Africa",
    subregion: "Middle Africa",
    population: 14497000,
    latlng: [15, 19],
    demonym: "Chadian",
    area: 1284000,
    gini: 39.8,
    density: 11,
    timezones: ["UTC+01:00"],
    borders: ["CMR", "CAF", "LBY", "NER", "NGA", "SSD"],
    nativeName: "Tchad",
    numericCode: "148",
    currencies: [
      {
        code: "XAF",
        name: "Central African CFA franc",
        symbol: "Fr"
      }
    ],
    languages: [
      {
        iso639_1: "fr",
        iso639_2: "fra",
        name: "French",
        nativeName: "français"
      },
      {
        iso639_1: "ar",
        iso639_2: "ara",
        name: "Arabic",
        nativeName: "العربية"
      }
    ],
    translations: {
      de: "Tschad",
      es: "Chad",
      fr: "Tchad",
      ja: "チャド",
      it: "Ciad",
      br: "Chade",
      pt: "Chade",
      nl: "Tsjaad",
      hr: "Čad",
      fa: "چاد"
    },
    flag: "https://restcountries.eu/data/tcd.svg",
    regionalBlocs: [
      {
        acronym: "AU",
        name: "African Union",
        otherAcronyms: [],
        otherNames: [
          "الاتحاد الأفريقي",
          "Union africaine",
          "União Africana",
          "Unión Africana",
          "Umoja wa Afrika"
        ]
      }
    ],
    cioc: "CHA"
  }
];

export const countriesSortedByNameAscMock = [
  {
    name: "Algeria",
    topLevelDomain: [".dz"],
    alpha2Code: "DZ",
    alpha3Code: "DZA",
    callingCodes: ["213"],
    capital: "Algiers",
    altSpellings: ["DZ", "Dzayer", "Algérie"],
    region: "Africa",
    subregion: "Northern Africa",
    population: 40400000,
    latlng: [28, 3],
    demonym: "Algerian",
    area: 2381741,
    gini: 35.3,
    timezones: ["UTC+01:00"],
    borders: ["TUN", "LBY", "NER", "ESH", "MRT", "MLI", "MAR"],
    nativeName: "الجزائر",
    numericCode: "012",
    currencies: [
      {
        code: "DZD",
        name: "Algerian dinar",
        symbol: "د.ج"
      }
    ],
    languages: [
      {
        iso639_1: "ar",
        iso639_2: "ara",
        name: "Arabic",
        nativeName: "العربية"
      }
    ],
    translations: {
      de: "Algerien",
      es: "Argelia",
      fr: "Algérie",
      ja: "アルジェリア",
      it: "Algeria",
      br: "Argélia",
      pt: "Argélia",
      nl: "Algerije",
      hr: "Alžir",
      fa: "الجزایر"
    },
    flag: "https://restcountries.eu/data/dza.svg",
    regionalBlocs: [
      {
        acronym: "AU",
        name: "African Union",
        otherAcronyms: [],
        otherNames: [
          "الاتحاد الأفريقي",
          "Union africaine",
          "União Africana",
          "Unión Africana",
          "Umoja wa Afrika"
        ]
      }
    ],
    cioc: "ALG"
  },
  {
    name: "Botswana",
    topLevelDomain: [".bw"],
    alpha2Code: "BW",
    alpha3Code: "BWA",
    callingCodes: ["267"],
    capital: "Gaborone",
    altSpellings: ["BW", "Republic of Botswana", "Lefatshe la Botswana"],
    region: "Africa",
    subregion: "Southern Africa",
    population: 2141206,
    latlng: [-22, 24],
    demonym: "Motswana",
    area: 582000,
    gini: 61,
    density: 4,
    timezones: ["UTC+02:00"],
    borders: ["NAM", "ZAF", "ZMB", "ZWE"],
    nativeName: "Botswana",
    numericCode: "072",
    currencies: [
      {
        code: "BWP",
        name: "Botswana pula",
        symbol: "P"
      }
    ],
    languages: [
      {
        iso639_1: "en",
        iso639_2: "eng",
        name: "English",
        nativeName: "English"
      },
      {
        iso639_1: "tn",
        iso639_2: "tsn",
        name: "Tswana",
        nativeName: "Setswana"
      }
    ],
    translations: {
      de: "Botswana",
      es: "Botswana",
      fr: "Botswana",
      ja: "ボツワナ",
      it: "Botswana",
      br: "Botsuana",
      pt: "Botsuana",
      nl: "Botswana",
      hr: "Bocvana",
      fa: "بوتسوانا"
    },
    flag: "https://restcountries.eu/data/bwa.svg",
    regionalBlocs: [
      {
        acronym: "AU",
        name: "African Union",
        otherAcronyms: [],
        otherNames: [
          "الاتحاد الأفريقي",
          "Union africaine",
          "União Africana",
          "Unión Africana",
          "Umoja wa Afrika"
        ]
      }
    ],
    cioc: "BOT"
  },
  {
    name: "Chad",
    topLevelDomain: [".td"],
    alpha2Code: "TD",
    alpha3Code: "TCD",
    callingCodes: ["235"],
    capital: "N'Djamena",
    altSpellings: ["TD", "Tchad", "Republic of Chad", "République du Tchad"],
    region: "Africa",
    subregion: "Middle Africa",
    population: 14497000,
    latlng: [15, 19],
    demonym: "Chadian",
    area: 1284000,
    gini: 39.8,
    density: 11,
    timezones: ["UTC+01:00"],
    borders: ["CMR", "CAF", "LBY", "NER", "NGA", "SSD"],
    nativeName: "Tchad",
    numericCode: "148",
    currencies: [
      {
        code: "XAF",
        name: "Central African CFA franc",
        symbol: "Fr"
      }
    ],
    languages: [
      {
        iso639_1: "fr",
        iso639_2: "fra",
        name: "French",
        nativeName: "français"
      },
      {
        iso639_1: "ar",
        iso639_2: "ara",
        name: "Arabic",
        nativeName: "العربية"
      }
    ],
    translations: {
      de: "Tschad",
      es: "Chad",
      fr: "Tchad",
      ja: "チャド",
      it: "Ciad",
      br: "Chade",
      pt: "Chade",
      nl: "Tsjaad",
      hr: "Čad",
      fa: "چاد"
    },
    flag: "https://restcountries.eu/data/tcd.svg",
    regionalBlocs: [
      {
        acronym: "AU",
        name: "African Union",
        otherAcronyms: [],
        otherNames: [
          "الاتحاد الأفريقي",
          "Union africaine",
          "União Africana",
          "Unión Africana",
          "Umoja wa Afrika"
        ]
      }
    ],
    cioc: "CHA"
  }
];

export const countriesSortedByNameDescMock = [
  {
    name: "Chad",
    topLevelDomain: [".td"],
    alpha2Code: "TD",
    alpha3Code: "TCD",
    callingCodes: ["235"],
    capital: "N'Djamena",
    altSpellings: ["TD", "Tchad", "Republic of Chad", "République du Tchad"],
    region: "Africa",
    subregion: "Middle Africa",
    population: 14497000,
    latlng: [15, 19],
    demonym: "Chadian",
    area: 1284000,
    gini: 39.8,
    density: 11,
    timezones: ["UTC+01:00"],
    borders: ["CMR", "CAF", "LBY", "NER", "NGA", "SSD"],
    nativeName: "Tchad",
    numericCode: "148",
    currencies: [
      {
        code: "XAF",
        name: "Central African CFA franc",
        symbol: "Fr"
      }
    ],
    languages: [
      {
        iso639_1: "fr",
        iso639_2: "fra",
        name: "French",
        nativeName: "français"
      },
      {
        iso639_1: "ar",
        iso639_2: "ara",
        name: "Arabic",
        nativeName: "العربية"
      }
    ],
    translations: {
      de: "Tschad",
      es: "Chad",
      fr: "Tchad",
      ja: "チャド",
      it: "Ciad",
      br: "Chade",
      pt: "Chade",
      nl: "Tsjaad",
      hr: "Čad",
      fa: "چاد"
    },
    flag: "https://restcountries.eu/data/tcd.svg",
    regionalBlocs: [
      {
        acronym: "AU",
        name: "African Union",
        otherAcronyms: [],
        otherNames: [
          "الاتحاد الأفريقي",
          "Union africaine",
          "União Africana",
          "Unión Africana",
          "Umoja wa Afrika"
        ]
      }
    ],
    cioc: "CHA"
  },
  {
    name: "Botswana",
    topLevelDomain: [".bw"],
    alpha2Code: "BW",
    alpha3Code: "BWA",
    callingCodes: ["267"],
    capital: "Gaborone",
    altSpellings: ["BW", "Republic of Botswana", "Lefatshe la Botswana"],
    region: "Africa",
    subregion: "Southern Africa",
    population: 2141206,
    latlng: [-22, 24],
    demonym: "Motswana",
    area: 582000,
    gini: 61,
    density: 4,
    timezones: ["UTC+02:00"],
    borders: ["NAM", "ZAF", "ZMB", "ZWE"],
    nativeName: "Botswana",
    numericCode: "072",
    currencies: [
      {
        code: "BWP",
        name: "Botswana pula",
        symbol: "P"
      }
    ],
    languages: [
      {
        iso639_1: "en",
        iso639_2: "eng",
        name: "English",
        nativeName: "English"
      },
      {
        iso639_1: "tn",
        iso639_2: "tsn",
        name: "Tswana",
        nativeName: "Setswana"
      }
    ],
    translations: {
      de: "Botswana",
      es: "Botswana",
      fr: "Botswana",
      ja: "ボツワナ",
      it: "Botswana",
      br: "Botsuana",
      pt: "Botsuana",
      nl: "Botswana",
      hr: "Bocvana",
      fa: "بوتسوانا"
    },
    flag: "https://restcountries.eu/data/bwa.svg",
    regionalBlocs: [
      {
        acronym: "AU",
        name: "African Union",
        otherAcronyms: [],
        otherNames: [
          "الاتحاد الأفريقي",
          "Union africaine",
          "União Africana",
          "Unión Africana",
          "Umoja wa Afrika"
        ]
      }
    ],
    cioc: "BOT"
  },
  {
    name: "Algeria",
    topLevelDomain: [".dz"],
    alpha2Code: "DZ",
    alpha3Code: "DZA",
    callingCodes: ["213"],
    capital: "Algiers",
    altSpellings: ["DZ", "Dzayer", "Algérie"],
    region: "Africa",
    subregion: "Northern Africa",
    population: 40400000,
    latlng: [28, 3],
    demonym: "Algerian",
    area: 2381741,
    gini: 35.3,
    timezones: ["UTC+01:00"],
    borders: ["TUN", "LBY", "NER", "ESH", "MRT", "MLI", "MAR"],
    nativeName: "الجزائر",
    numericCode: "012",
    currencies: [
      {
        code: "DZD",
        name: "Algerian dinar",
        symbol: "د.ج"
      }
    ],
    languages: [
      {
        iso639_1: "ar",
        iso639_2: "ara",
        name: "Arabic",
        nativeName: "العربية"
      }
    ],
    translations: {
      de: "Algerien",
      es: "Argelia",
      fr: "Algérie",
      ja: "アルジェリア",
      it: "Algeria",
      br: "Argélia",
      pt: "Argélia",
      nl: "Algerije",
      hr: "Alžir",
      fa: "الجزایر"
    },
    flag: "https://restcountries.eu/data/dza.svg",
    regionalBlocs: [
      {
        acronym: "AU",
        name: "African Union",
        otherAcronyms: [],
        otherNames: [
          "الاتحاد الأفريقي",
          "Union africaine",
          "União Africana",
          "Unión Africana",
          "Umoja wa Afrika"
        ]
      }
    ],
    cioc: "ALG"
  }
];
