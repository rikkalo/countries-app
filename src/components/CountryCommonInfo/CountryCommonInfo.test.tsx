import { shallow } from "enzyme";
import React from "react";

import { countryMock } from "../../mock";
import { CountryCommonInfo } from "./index";

describe("CountryCommonInfo", () => {
  it("should render correctly", () => {
    const wrapper = shallow(<CountryCommonInfo country={countryMock} />);

    expect(wrapper).toMatchSnapshot();
  });
});
