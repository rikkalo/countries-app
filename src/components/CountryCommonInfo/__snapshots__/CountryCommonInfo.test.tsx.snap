// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CountryCommonInfo should render correctly 1`] = `
<WithStyles(ForwardRef(Grid))
  container={true}
  spacing={3}
>
  <WithStyles(ForwardRef(Grid))
    item={true}
    md={12}
    xs={12}
  >
    <WithStyles(ForwardRef(Typography))
      component="h3"
      variant="h3"
    >
      Botswana
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    item={true}
    md={6}
    xs={12}
  >
    <Flag
      name="Botswana"
      value="https://restcountries.eu/data/bwa.svg"
    />
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    item={true}
    md={6}
    xs={12}
  >
    <div>
      <WithStyles(ForwardRef(Typography))
        className="title"
        color="textPrimary"
        component="span"
        variant="subtitle1"
      >
        Top level domain:
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        component="span"
        variant="subtitle2"
      >
        .bw
      </WithStyles(ForwardRef(Typography))>
    </div>
    <div>
      <WithStyles(ForwardRef(Typography))
        className="title"
        color="textPrimary"
        component="span"
        variant="subtitle1"
      >
        Coordinates:
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        component="span"
        variant="subtitle2"
      >
        -22
        , 
        24
      </WithStyles(ForwardRef(Typography))>
    </div>
    <div
      key="ISO 3166-1 alpha-2"
    >
      <WithStyles(ForwardRef(Typography))
        className="title"
        component="span"
        variant="subtitle1"
      >
        ISO 3166-1 alpha-2
        :
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        component="span"
        variant="subtitle2"
      >
        BW
      </WithStyles(ForwardRef(Typography))>
    </div>
    <div
      key="ISO 3166-1 alpha-3"
    >
      <WithStyles(ForwardRef(Typography))
        className="title"
        component="span"
        variant="subtitle1"
      >
        ISO 3166-1 alpha-3
        :
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        component="span"
        variant="subtitle2"
      >
        BWA
      </WithStyles(ForwardRef(Typography))>
    </div>
    <div
      key="Capital"
    >
      <WithStyles(ForwardRef(Typography))
        className="title"
        component="span"
        variant="subtitle1"
      >
        Capital
        :
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        component="span"
        variant="subtitle2"
      >
        Gaborone
      </WithStyles(ForwardRef(Typography))>
    </div>
    <div
      key="Region"
    >
      <WithStyles(ForwardRef(Typography))
        className="title"
        component="span"
        variant="subtitle1"
      >
        Region
        :
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        component="span"
        variant="subtitle2"
      >
        Africa
      </WithStyles(ForwardRef(Typography))>
    </div>
    <div
      key="Subregion"
    >
      <WithStyles(ForwardRef(Typography))
        className="title"
        component="span"
        variant="subtitle1"
      >
        Subregion
        :
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        component="span"
        variant="subtitle2"
      >
        Southern Africa
      </WithStyles(ForwardRef(Typography))>
    </div>
    <div
      key="Population"
    >
      <WithStyles(ForwardRef(Typography))
        className="title"
        component="span"
        variant="subtitle1"
      >
        Population
        :
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        component="span"
        variant="subtitle2"
      >
        2141206
      </WithStyles(ForwardRef(Typography))>
    </div>
    <div
      key="Demonym"
    >
      <WithStyles(ForwardRef(Typography))
        className="title"
        component="span"
        variant="subtitle1"
      >
        Demonym
        :
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        component="span"
        variant="subtitle2"
      >
        Motswana
      </WithStyles(ForwardRef(Typography))>
    </div>
    <div
      key="Native name"
    >
      <WithStyles(ForwardRef(Typography))
        className="title"
        component="span"
        variant="subtitle1"
      >
        Native name
        :
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        component="span"
        variant="subtitle2"
      >
        Botswana
      </WithStyles(ForwardRef(Typography))>
    </div>
    <div
      key="Numeric code"
    >
      <WithStyles(ForwardRef(Typography))
        className="title"
        component="span"
        variant="subtitle1"
      >
        Numeric code
        :
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        component="span"
        variant="subtitle2"
      >
        072
      </WithStyles(ForwardRef(Typography))>
    </div>
    <div
      key="Area"
    >
      <WithStyles(ForwardRef(Typography))
        className="title"
        component="span"
        variant="subtitle1"
      >
        Area
        :
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        component="span"
        variant="subtitle2"
      >
        582000
      </WithStyles(ForwardRef(Typography))>
    </div>
    <div
      key="Gini coefficient"
    >
      <WithStyles(ForwardRef(Typography))
        className="title"
        component="span"
        variant="subtitle1"
      >
        Gini coefficient
        :
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        component="span"
        variant="subtitle2"
      >
        61
      </WithStyles(ForwardRef(Typography))>
    </div>
    <div
      key="Cioc"
    >
      <WithStyles(ForwardRef(Typography))
        className="title"
        component="span"
        variant="subtitle1"
      >
        Cioc
        :
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        component="span"
        variant="subtitle2"
      >
        BOT
      </WithStyles(ForwardRef(Typography))>
    </div>
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Grid))>
`;
